desktop-widgets/tab-widgets/maintab.cpp

```diff
@@ -386,7 +386,7 @@ void MainTab::showLocation()
 // has a parameter on it's definition it didn't worked on the signal slot connection.
 void MainTab::refreshDiveInfo()
 {
-	updateDiveInfo();
+	MainWindow::instance()->refreshDisplay();
```

Comment:
This is one of those tab-widgets calling MainWindow things that @tcanabrava hates and is trying to clean up.

I will take it since it fixes a bug, but should we maybe turn this into a signal/slot situation instead of a direct call like this?